LICENSE

```diff
@@ -1,6 +1,10 @@
-The MIT License (MIT)
+This license applies to everything except the contents of the "test"
+directory and its subdirectories.
 
-Copyright (c) 2017 Huu Nguyen
+MIT License
+
+Copyright (c) 2017-2021 Huu Nguyen
+Copyright (c) 2021 Fabian Sauter
```

Comment:
I think the copyright should belong to libcpr not you. But at the end of the day I am not a Layer 🤷.

Reply:
Agree.
Based on: https://fairuse.stanford.edu/overview/faqs/copyright-ownership/
and: http://haacked.com/archive/2006/01/26/WhoOwnstheCopyrightforAnOpenSourceProject.aspx/

Since we do not have any statement when people are contributing, stating something like „I hereby assign copyright in this code to the project, to be licensed under the same terms as the rest of the code.“ the copyright for their code stays with them.

So we should change it to something like: „libcpr and many other contributors“. This should be fine, since the term „libcpr“ refers indirectly to us maintainers and all other contributors.

Also I don‘t think switching away from this dual license will be possible any time soon, since every copyright owner has to agree with this change.